### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by marians**
Applied some conventions to improve rendering and readability.

RFR @teemow

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -13,34 +13,34 @@ For now Onsho only supports QEMU/KVM. Which makes Linux a requirement. But you s
 
 ## Building Onsho
 
-```
-git clone https://github.com/giantswarm/onsho.git
-cd onsho
-make && sudo make install
+```nohighlight
+$ git clone https://github.com/giantswarm/onsho.git
```
Comment (teemow) [inline docs/getting_started.md]: afaik we shouldn't add the `$` signs. This makes it impossible to copy paste the snippets.

**Comment by teemow** [comment]
LGTM besides the comment

**Comment by hectorj2f** [comment]
@marians :D ... are we ready to merge this PR ?

**Comment by marians** [comment]
Removed the `$` signs to make @teemow happy. Merging.

**Pull request merged**